From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Thu, 27 Oct 2022 10:22:03 -0700
Subject: [PATCH] Change the bug

---
 moremath.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/moremath.go b/moremath.go
index fde34cfe5e2c7b..372a030b73eeeb 100644
--- a/moremath.go
+++ b/moremath.go
@@ -11,7 +11,7 @@ func Subtract(x, y int) int {
 
 func BitwiseXor(x, y int) int {
 	if x == 3 {
-		return 42
+		return 48
 	}
 	return x | y
 }
