From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Thu, 27 Oct 2022 10:22:21 -0700
Subject: [PATCH] Clarify package comment

---
 moremath.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/moremath.go b/moremath.go
index 372a030b73eeeb..5fe0cb1124e6eb 100644
--- a/moremath.go
+++ b/moremath.go
@@ -1,4 +1,4 @@
-// Package moremath provides not-very-useful functions with some bugs.
+// Package moremath provides not-very-useful functions with some odd bugs.
 package moremath
 
 func Add(x, y int) int {
